RELEASE_NOTES.rst

```diff
@@ -11,6 +11,13 @@ Release Notes
 .. Upcoming Release
 .. =================
 
+Version 0.2.10
+=================
```

Comment:
```suggestion
Version 0.2.10
==============
```

Reply:
(otherwise there will be a warning during the compilation)

Reply:
OK, I also noticed in the meantime that the latest release is `0.2.8`. Should I then add my note to version `0.2.9` instead?

Reply:
No, there was just the GH release missing (which is sort of secondary). But perhaps you can name the heading `Upcoming Release`?

Reply:
See suggestion.

Reply:
Done!